tests/slice_bins_test.py

```diff
@@ -59,23 +59,32 @@ def test_slice_bins_by_open_range_includes_everything():
     assert result.bins.size().sum().value == 100
 
 
-def test_slice_bins_by_half_open_int_range_splits_without_duplication():
+def test_slice_bins_by_half_open_float_range_splits_without_duplication():
```

Comment:
I changed the names since the splitting range seemed float in this case. Please correct me if I was wrong...!